cmd/zetaclientd/start_utils.go

```diff
@@ -13,16 +13,13 @@ import (
 	"google.golang.org/grpc"
 )
 
-func setMYIP(cfg *config.Config, logger zerolog.Logger) {
-	if os.Getenv("MYIP") == "" {
-		if cfg.PublicIP == "" {
-			logger.Fatal().Msg("Please set MYIP environment variable or use the PublicIP flag")
-		}
-		err := os.Setenv("MYIP", cfg.PublicIP)
-		if err != nil {
-			logger.Fatal().Err(err).Msg("Error setting MYIP")
-		}
+func getMYIP(logger zerolog.Logger) (string, error) {
+	ip := os.Getenv("MYIP")
+	if ip == "" {
+		logger.Fatal().Msg("Please set MYIP environment variable")
+		return "", errors.New("empty ip address")
 	}
```

Comment:
why remove setting MYIP via config ?

Reply:
I think we need to choose one, either the env variable or the config file. It is confusing when we tried to debug whether the public IP was set correctly in each node.